antenna/breakpad_resource.py

```diff
@@ -206,6 +206,9 @@ def extract_payload(self, req):
                     # of "malformed JSON" situations.
                     raise MalformedCrashReport("bad_json")
 
+                if not isinstance(raw_crash, dict):
+                    raise MalformedCrashReport("bad_json")
```

Comment:
We might want to enforce a schema here at some point, but I'm not ready to do that, yet.